frontend/src/layout/panel-layout/ProjectTree/projectTreeLogic.tsx

```diff
@@ -112,6 +114,32 @@ export const projectTreeLogic = kea<projectTreeLogicType>([
                 },
             },
         ],
+        archivedFolder: [
+            false as boolean,
+            {
+                loadArchivedFolder: async () => {
+                    try {
+                        const response = await api.fileSystem.list({ path: 'Archived' })
+                        if (response.results.length === 0) {
+                            await api.fileSystem.create({
+                                id: PROJECT_TREE_ARCHIVE_NAME,
+                                path: PROJECT_TREE_ARCHIVE_NAME,
+                                type: 'folder',
+                            })
+                        }
+                        actions.loadFolder(PROJECT_TREE_ARCHIVE_NAME)
+                        for (const folder of Object.keys(values.folders)) {
+                            if (folder.startsWith(PROJECT_TREE_ARCHIVE_NAME + '/')) {
+                                actions.loadFolder(folder)
+                            }
+                        }
```

Comment:
If the archive folder isn't loaded yet, then I don't think this code immediately after will do anything. I'm not sure we really need to load all the contents of all the inner folders.